akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/RetentionCriteria.scala

```diff
@@ -5,8 +5,11 @@
 package akka.persistence.typed.scaladsl
 
 import akka.annotation.DoNotInherit
-import akka.persistence.typed.internal.DisabledRetentionCriteria
-import akka.persistence.typed.internal.SnapshotCountRetentionCriteriaImpl
+import akka.persistence.typed.internal.{
+  DisabledRetentionCriteria,
+  SnapshotCountRetentionCriteriaImpl,
+  SnapshotPredicateCriteriaImpl
+}
```

Comment:
General Akka umbrella import style is one line per import (avoids some git import conflicts)